test/classes/errors/nilability-init-var/init-var-nonnil-owned-from-nonnil-borrowed.good

```diff
@@ -0,0 +1 @@
+init-var-nonnil-owned-from-nonnil-borrowed.chpl:9: error: cannot assign to an owned variable from a borrowed class
```

Comment:
This error message makes it sound like the offending code had `lhs = borrowed MyClass`